src/gl.rs

```diff
@@ -160,8 +160,8 @@ pub trait Gl {
     fn active_texture(&self, texture: GLenum);
     fn attach_shader(&self, program: GLuint, shader: GLuint);
     fn bind_attrib_location(&self, program: GLuint, index: GLuint, name: &str);
-    fn get_uniform_iv(&self, program: GLuint, location: GLint) -> Vec<GLint>;
-    fn get_uniform_fv(&self, program: GLuint, location: GLint) -> Vec<GLfloat>;
+    fn get_uniform_iv(&self, program: GLuint, location: GLint, uniform_type: GLuint) -> Vec<GLint>;
```

Comment:
hmm. One problem is - this is a breaking change. Another - the new signature no longer matches GL.

Reply:
Luckily, it is not yet used anywhere.
To preserve the signature, we could alternatively call `get_active_uniform(program, location)` within gleam to get the uniform type instead of passing it. What do you think?

Reply:
I'd rather have us calling `get_active_uniform` then. Yes, it's slower, but we don't expect any hot paths going through `get_uniform_iv` anyway.

Reply:
Alright, updated the PR.